autogen/oai/client.py

```diff
@@ -425,6 +431,8 @@ def _register_default_client(self, config: Dict[str, Any], openai_config: Dict[s
                 self._configure_azure_openai(config, openai_config)
                 client = AzureOpenAI(**openai_config)
                 self._clients.append(OpenAIClient(client))
+            elif api_type is not None and api_type.startswith("google"):
+                self._clients.append(GeminiClient(**openai_config))
```

Comment:
Can we throw an exception here if the Gemini import unsuccessfully due to dependency not installed? Also to remind user to install the packages.